x11-misc/light-locker/light-locker-1.8.0-r1.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools gnome2-utils
+
+DESCRIPTION="A simple locker using lightdm"
+HOMEPAGE="https://github.com/the-cavalry/light-locker"
+SRC_URI="${HOMEPAGE}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2 LGPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+consolekit +dpms +elogind gtk3 +screensaver -systemd +upower"
+
+RDEPEND="dev-libs/dbus-glib
+	>=dev-libs/glib-2.25.6:2
```

Comment:
This version requirement can be dropped, since the latest version of glib in gentoo tree is 2.56.